package-lock.json

```diff
@@ -3,10 +3,36 @@
   "requires": true,
   "lockfileVersion": 1,
   "dependencies": {
+    "@babel/code-frame": {
+      "version": "7.12.13",
+      "resolved": "https://artifactory.corp.creditkarma.com:8443/api/npm/npmjs/@babel/code-frame/-/code-frame-7.12.13.tgz",
```

Comment:
This file references https://artifactory.corp.creditkarma.com:8443 and taht hostname doesn't resolve for me. I don't know if that's because it's some kind of internal-only DNS or if it no longer exists.

And I don't know if that's a problem. Even if it _is_ a problem it doesn't seem terrible--we could just recreate the lock file. Either as part of this PR or in a follow-up change.

Reply:
Yeah, that's pretty weird - when I did an `npm install` on the repo, everything worked for me. So, I think I'll just merge it as-is and then recreate the `package-lock.json` file in another PR.